provider/testdata/data/json/datetime/gregory_patterns@1/ar-EG.json

```diff
@@ -18,50 +18,10 @@
     "short": "HH:mm"
   },
   "preferred_hour_cycle": "H11H12",
-  "datetime": {
```

Comment:
Thought: I'm not a fan of this being called `gregory_patterns` now. These are specifically "length" based patterns. It's ambiguous in regards to other data providers which also have patterns.